src/color/setting.js

```diff
@@ -11,25 +11,36 @@ import * as constants from '../core/constants';
 import './p5.Color';
 
 /**
```

Comment:
@zelf0 you're off to a great start here. What do you think about the following revisions to the prose?

Sets the color used for the background of the canvas. By default, the background is transparent. This function is typically used within <a href="#/p5/draw">draw()</a> to clear the display window at the beginning of each frame. It can also be used inside <a href="#/p5/setup">setup()</a> to set the background on the first frame of animation.

The version of `background()` with one parameter interprets the value one of four ways. If the parameter is a number, it's interpreted as a grayscale value. If the parameter is a string, it's interpreted as a CSS color string. RGB, RGBA, HSL, HSLA, hex, and named color strings are supported. If the parameter is a <a href="#/p5.Color">p5.Color</a> object, it will be used as the background color. If the parameter is a <a href="#/p5.Image">p5.Image</a> object, it will be used as the background image.

The version of `background()` with two parameters interprets the first one as a grayscale value. The second parameter sets the alpha (transparency) value.

The version of `background()` with three parameters interprets them as RGB, HSB, or HSL colors, depending on the current <a href="#/p5/colorMode">colorMode()</a>. By default, colors are specified in RGB values. Calling `background(255, 204, 0)` sets the background a bright yellow color.

The version of `background()` with four parameters interprets them as RGBA, HSBA, or HSLA colors, depending on the current <a href="#/p5/colorMode">colorMode()</a>. By default, colors are specified in RGBA values. Calling `background(255, 204, 0, 20)` sets the background a transparent, bright yellow color.

Reply:
Yeah, that could be more consistent with the other color docs and it's not necessary to specify multiple times that the parameters of background are RGB so I like that you took that part out.

Reply:
Committed the changes.